frontend/src/pages/notebookController/screens/server/useAcceleratorProfiles.ts

```diff
@@ -5,7 +5,13 @@ import { listAcceleratorProfiles } from '~/api';
 
 const useAcceleratorProfiles = (namespace: string): FetchState<AcceleratorProfileKind[]> => {
   const getAcceleratorProfiles = React.useCallback(
-    () => listAcceleratorProfiles(namespace),
+    () =>
+      listAcceleratorProfiles(namespace).catch((e) => {
+        if (e.statusObject?.code === 404) {
+          throw new Error('No accelerators profiles found.');
+        }
```

Comment:
Fairly certain 404 never occurs with k8s list resources. It will always return an empty list.
I see that `useGroups` does this but it is likely dead code.
You can remove this change as well as the related tests.